css/README.md

```diff
@@ -410,3 +413,47 @@ When using HSL or RGB notation, always add a single space after a comma (,) and
 We prefer using mixins over functions. If a function is absolutely necessary, document where it is used and why it wasn't possible to create a mixin.
 
 **[⬆ back to top](#table-of-contents)**
+
+## CSS Modules
+
+[CSS Modules documentation](https://github.com/css-modules/css-modules)
+
+- Prefer using CSS Modules over global css code
+- Use camelCase for class names
+- Always use `.module.css` or `.module.scss` extension
+- Do not expect any scss variables or mixins to be defined. You have to import them yourself.
+- Prefer Sass variables over CSS Module variables
+- Do not use CSS Modules composition feature. Use Sass mixins instead.
```

Comment:
Composition feature is almost the same as Sass' extend feature. Since there are good reasons to stay away from sass-extend, we should do the same for css-modules-composition.

Reply:
agreed, pretty bad. we should use mixins instead. css bloat is not that big.